### PR TITLE
Social authentication

### PR DESCRIPTION
**Pull request by ilya-github**

### CODE REVIEW

File: config/database.yml.example [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-  database: dasha_taxpayer_test
-
-production:
-  <<: *default
-  database: dasha_taxpayer_production
-  username: dasha_taxpayer
-  password: <%= ENV['TEST_PG_DATABASE_PASSWORD'] %>
-
-cucumber:
-  <<: *test
```
Comment (VDmitryO) [inline config/database.yml.example]: return this file

File: config/secrets.yml.example [hunk 2]
```diff
@@ -1,3 +0,0 @@
-development:
-  mail_login: 'example@gmail.com'
-  mail_password: 'password'
```
Comment (VDmitryO) [inline config/secrets.yml.example]: return this file and add examples of keys for social networks

**Comment by allomov** [comment]
1. tests do not work.
2. there is no test environment in secrets.yml
2. where do you have a link to sign in with github?

**Comment by allomov** [comment]
+ merge with master

**Pull request merged**